public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src ws://localhost:3000 http://localhost:3000 ws://127.0.0.1:9944 wss://*.cere.network:9945 https://*.infura.io https://*.google-analytics.com wss://*.walletconnect.org https://*.walletconnect.com https://*.sentry.io; style-src 'self' 'unsafe-inline'; img-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.googletagmanager.com">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src ws://localhost:3000 http://localhost:3000 ws://127.0.0.1:9944 wss://*.cere.network https://*.infura.io https://*.google-analytics.com wss://*.walletconnect.org https://*.walletconnect.com https://*.sentry.io https://*.etherchain.org; style-src 'self' 'unsafe-inline'; img-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.googletagmanager.com">
```

Comment:
What is `*etherchain.org`?

Reply:
This a fallback request to `https://www.etherchain.org/api/gasPriceOracle` to get the gas price called when an Infura node is not available.